components/error/appBoundary/src/index.js

```diff
@@ -3,10 +3,24 @@ import PropTypes from 'prop-types'
 
 class ErrorAppBoundary extends Component {
   AlertBasicComponent = null
-  state = { hasError: false }
+  state = { errorCount: 0, hasError: false }
   componentDidCatch (errorMessage, errorStack) {
-    this.props.onError({ errorMessage, errorStack })
-    // import async the alert basic component
+    const { errorThreshold, onError, redirectUrlOnBreakingThreshold } = this.props
+    const { errorCount } = this.state
+
+    onError({ errorMessage, errorStack })
+    this.setState({ errorCount: errorCount + 1 })
+
+    return errorCount >= errorThreshold
+      ? redirectUrlOnBreakingThreshold && (window.location.href = redirectUrlOnBreakingThreshold)
```

Comment:
Does the "componentDidCatch" occurs only in client side?

Reply:
If so, then it's ok, but if dont', this window.location.href would fail in server side

Reply:
componentDidCatch dont' catch errors for server side rendering.

https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries

Reply:
great! thx :)

Reply:
Thanks to you!